modules/RTC/RTCUtils.js

```diff
@@ -114,9 +114,14 @@ function getConstraints(um, options) {
         constraints.video = { mandatory: {}, optional: [] };
 
         if (options.cameraDeviceId) {
-            // new style of settings device id (FF only)
-            constraints.video.deviceId = options.cameraDeviceId;
-            // old style
+            // Don't mix new and old style settings for Chromium as this leads
+            // to TypeError in new Chromium versions. @see
+            // https://bugs.chromium.org/p/chromium/issues/detail?id=614716
+            if (!RTCBrowserType.isChrome() && !RTCBrowserType.isOpera()) {
```

Comment:
> So the fix applies to Chrome and Opera only? Not React Native?
It applies for Chromium-based devices only.

> Since you have the check 3 times, could I please ask you to have it once stored in a variable with a descriptive name which makes it clear that it stands for a new style of constraints or something like that?

Fixed